x-pack/legacy/plugins/ml/public/components/navigation_menu/tabs.tsx

```diff
@@ -17,12 +17,8 @@ interface Props {
   tabId: TabId;
 }
 
-function getTabs(tabId: TabId, disableLinks: boolean): Tab[] {
-  const TAB_MAP: Record<TabId, Tab[]> = {
-    // overview: [],
-    datavisualizer: [],
-    data_frames: [],
-    data_frame_analytics: [],
+export function getTabs(tabId: TabId, disableLinks: boolean): Tab[] {
+  const TAB_MAP: Partial<Record<TabId, Tab[]>> = {
```

Comment:
i quite liked having the other main tabs listed here, even though they are empty, because it explicitly states that those tabs have no sub tabs.

Reply:
Added back in a90916d.